osu.Game.Tests/Visual/Gameplay/TestSceneReplayRecording.cs

```diff
@@ -45,7 +47,11 @@ public TestSceneReplayRecording()
                     {
                         recordingManager = new TestRulesetInputManager(new TestSceneModSettings.TestRulesetInfo(), 0, SimultaneousBindingMode.Unique)
                         {
-                            Recorder = new TestReplayRecorder(new Score { Replay = replay })
+                            Recorder = new TestReplayRecorder(new Score
+                            {
+                                Replay = replay,
+                                ScoreInfo = { Beatmap = new TestBeatmap(new OsuRuleset().RulesetInfo).BeatmapInfo }
```

Comment:
Any reason for not using the already constructed beatmap? Seems to work okay (requires removing some using statements).

```suggestion
                                ScoreInfo = { Beatmap = gameplayBeatmap.BeatmapInfo }
```